src/stylus/fablevision/fv-chromosome.styl

```diff
@@ -15,24 +15,30 @@
     height: 122px
     width: 20px
 
+    &.one
+      background: url('../resources/fablevision/layout-c/chromosome_1.png')
```

Comment:
This occurs in many places, but I think `background-image` would be preferred here and elsewhere ([CSS Shorthand Syntax Considered an Anti-Pattern](https://csswizardry.com/2016/12/css-shorthand-syntax-considered-an-anti-pattern/)).